torch/utils/data/dataloader.py

```diff
@@ -230,6 +263,8 @@ def _set_SIGCHLD_handler():
         return
     previous_handler = signal.getsignal(signal.SIGCHLD)
     if not callable(previous_handler):
+        # This doesn't catch SIGDFL default handler, but SIGCHLD default handler
+        # is no-op.
```

Comment:
FWIW the default handler for SIGCHLD is not a no-op (it clears the zombie), but it doesn't matter that much I guess. Why do we care about SIGDFL?

Reply:
SIGDFL is the name of signal default handler `signal.SIGDFL`. :) So I was just talking about the default handler. I will make it clearer.

Reply:
I did a bit of a search and believe that the default action is "ignore".